cursorless-talon/src/positional_target.py

```diff
@@ -1,44 +0,0 @@
-from typing import Any
-
-from talon import Module
-
-from .modifiers.position import construct_positional_modifier
-
-mod = Module()
-
-
-@mod.capture(
-    rule=(
-        "({user.cursorless_position} | {user.cursorless_source_destination_connective}) "
-        "<user.cursorless_target>"
-    )
-)
-def cursorless_positional_target(m) -> dict[str, Any]:
-    target: dict[str, Any] = m.cursorless_target
-    try:
-        modifier = construct_positional_modifier(m.cursorless_position)
-        return update_first_primitive_target(target, modifier)
-    except AttributeError:
-        return target
-
-
-def update_first_primitive_target(target: dict[str, Any], modifier: dict[str, Any]):
```

Comment:
fine with removing for now, but as discussed in meet-up, we'll prob need this for some stuff people will want to do with custom spoken api

Reply:
Agreed. That code would also operate on our data classes and support target and destinations and so on. I definitely feel that that is a problem for future us :D